source/blog/2017-04-13-rpm-ostree-v2017.4-released.html.md

```diff
@@ -0,0 +1,234 @@
+directly update the booted deployment, so that rebooting is
+no longer needed.
+
+### RPMs can now be directly installed
+
+Previously, if you wanted to overlay a specific RPM file,
+you'd have to shove it into a repo and place that repo in
+`/etc/yum.repos.d`. With v2017.4, rpm-ostree is now able to
+install it directly:
+
+```
+# rpm-ostree install ltrace.rpm
+# rpm-ostree status
+State: idle
+Deployments:
+  fedora-atomic:fedora-atomic/25/x86_64/docker-host
+             Version: 25.89 (2017-03-26 21:04:50)
+          BaseCommit: 6a71adb06bc296c19839e951c38dc0b71ee5d7a82262fef9612f256f0c2a70da
+              OSName: fedora-atomic
+       LocalPackages: ltrace-0.7.91-20.fc25.x86_64
+
+* fedora-atomic:fedora-atomic/25/x86_64/docker-host
+             Version: 25.89 (2017-03-26 21:04:50)
+          BaseCommit: 6a71adb06bc296c19839e951c38dc0b71ee5d7a82262fef9612f256f0c2a70da
+              OSName: fedora-atomic
+```
+
+As you can see above, the package shows up in a new
+`LocalPackages`.  This is different from `LayeredPackages`:
+packages in the latter are updated as new deployments are
+created, whereas those in the former are *never* updated. So
+in the example above, `ltrace` is cemented at that version.
+
+You can always "unstick" a local package and fetch it back
+from the repos:
+
+```
+# rpm-ostree uninstall ltrace-0.7.91-20.fc25.x86_64 --install ltrace
+# rpm-ostree status
+State: idle
+Deployments:
+  fedora-atomic:fedora-atomic/25/x86_64/docker-host
+             Version: 25.89 (2017-03-26 21:04:50)
+          BaseCommit: 6a71adb06bc296c19839e951c38dc0b71ee5d7a82262fef9612f256f0c2a70da
+              OSName: fedora-atomic
+     LayeredPackages: ltrace
+
+* fedora-atomic:fedora-atomic/25/x86_64/docker-host
+             Version: 25.89 (2017-03-26 21:04:50)
+          BaseCommit: 6a71adb06bc296c19839e951c38dc0b71ee5d7a82262fef9612f256f0c2a70da
+              OSName: fedora-atomic
+       LocalPackages: ltrace-0.7.91-20.fc25.x86_64
+```
+
+### Configuration can now be reloaded (v2017.3)
+
+Previously, one had to restart the `rpm-ostreed` service to
+get it to notice new remotes or repo configuration changes.
+Starting from v2017.3, there is now a `rpm-ostree reload`
+command available to do this. System administrators might
+prefer to use the equivalent `systemctl reload rpm-ostreed`.
+
+### More powerful package layering semantics (v2017.3)
+
+rpm-ostree is now much smarter about how it handles layered
+RPMs. Previously, one couldn't add a package that already
+existed in the base layer. Additionally, previously layered
+packages would magically disappear if an update added that
+same package. If another update subsequently removed that
+package again, you'd be left without it layered at all.
+
+Starting in v2017.3, rpm-ostree treats layered packages in a
+more declarative manner; packages requested through the
+`install` command are interpreted as directives to ensure
+they are always present. They will then be layered only when
```

Comment:
Good point. I added a blurb at the end of the section. Didn't feel like redoing the printouts.